trestlebot/cli/base.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import logging
+from typing import List
+
+from trestlebot.bot import TrestleBot
+from trestlebot.cli.options.common import handle_exceptions
+from trestlebot.tasks.assemble_task import AssembleTask
+from trestlebot.tasks.authored import types
+from trestlebot.tasks.authored.base_authored import AuthoredObjectBase
+from trestlebot.tasks.base_task import ModelFilter, TaskBase
+from trestlebot.tasks.regenerate_task import RegenerateTask
+
+
+logger = logging.getLogger(__name__)
+
+
+def comma_sep_to_list(string: str) -> List[str]:
+    """Convert comma-sep string to list of strings and strip."""
+    string = string.strip() if string else ""
+    return list(map(str.strip, string.split(","))) if string else []
+
+
+def run_base(args: argparse.Namespace, pre_tasks: List[TaskBase]) -> None:
```

Comment:
@qduanmu could we drop the use of argparse here and instead just pass keyword arguments?  As part of the move to Click we are hoping to drop the use of argparse.